src/plib/resources/templates/plesk.com.web.json

```diff
@@ -10,16 +10,47 @@
     "variableDescription": "ip is the IP address of the server.",
     "records": [
         {
+            "groupId": "WebService",
             "type": "A",
             "host": "@",
             "pointsTo": "%ip%",
             "ttl": "600"
         },
         {
+            "groupId": "WebService",
             "type": "CNAME",
             "host": "www",
             "pointsTo": "@",
             "ttl": "600"
+        },
+        {
+            "groupId": "MailService",
+            "type": "MX",
+            "pointsTo": "%mxhost%",
```

Comment:
we always use `mail.` subdomain

Reply:
customer can use any hostname for his mail service